internal/app.go

```diff
@@ -232,6 +246,42 @@ OPTIONS:
 	return app
 }
 
+func showVersion(cacheDir, outputFormat, version string, outputWriter io.Writer) {
+	db.Init(cacheDir)
+	metadata, err := db.Config{}.GetMetadata()
+	if err != nil {
+		fmt.Fprintf(outputWriter, "unable to display current version: %s", err.Error())
+		return
+	}
+	switch outputFormat {
+	case "json":
+		b, _ := json.Marshal(ttypes.VersionInfo{
+			Trivy: version,
+			VulnerabilityDB: db.Metadata{
+				Version:    metadata.Version,
+				Type:       metadata.Type,
+				NextUpdate: metadata.NextUpdate.UTC(),
+				UpdatedAt:  metadata.UpdatedAt.UTC(),
+			},
+		})
+		fmt.Fprintln(outputWriter, string(b))
+	default:
+		table := tablewriter.NewWriter(outputWriter)
+		table.SetHeader([]string{"Component", "Version"})
+		table.Append([]string{"Trivy", version})
+		switch metadata.Type {
+		case 0:
+			table.Append([]string{"VulnDB Type", "Full"})
+		case 1:
+			table.Append([]string{"VulnDB Type", "Light"})
+		}
+		table.Append([]string{"VulnDB Version", strconv.Itoa(metadata.Version)})
+		table.Append([]string{"VulnDB Updated At", metadata.UpdatedAt.UTC().String()})
+		table.Append([]string{"VulnDB Next Update", metadata.NextUpdate.UTC().String()})
+		table.Render()
```

Comment:
I'm sorry I didn't mean a table format, but a plain text. Also, the current table is not easy to see.

```
+--------------------+--------------------------------+
|     COMPONENT      |            VERSION             |
+--------------------+--------------------------------+
| Trivy              | dev                            |
| VulnDB Type        | Light                          |
| VulnDB Version     |                              1 |
| VulnDB Updated At  | 2020-03-17 12:08:34.013998914  |
|                    | +0000 UTC                      |
| VulnDB Next Update | 2020-03-18 00:08:34.013998514  |
|                    | +0000 UTC                      |
+--------------------+--------------------------------+
```

I think `docker version` is preferred here.

```
Version: v0.5.2
Vulnerability DB:
  Type: Light
  Version: 1
  UpdatedAt: 2020-03-17 12:08:34.013998914 +0000 UTC
  NextUpdate: 2020-03-18 00:08:34.013998514 +0000 UTC 
```

I know the default value of `-format` is `table`, but I think we can ignore it in this case.

Reply:
Fixed here https://github.com/aquasecurity/trivy/pull/435/commits/6e4f2db837deb3ce456cf6a2776d8c01128ae60d